scripts/qa_crowbarsetup.sh

```diff
@@ -4525,6 +4525,10 @@ function onadmin_prepare_crowbar_upgrade
         # move nodes to upgrade mode
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
+        if ! crowbarctl upgrade repocheck crowbar; then
+            complain 11 "Error running 'crowbarctl upgrade repocheck crowbar'"
+        fi
```

Comment:
oh, just read the second comment. I would prefer not to have a temporary file just for this, since we would need to delete it afterwards. I think this approach, although not perfect, looks cleaner.